pkg/kubelet/types/labels.go

```diff
@@ -21,6 +21,7 @@ const (
 	KubernetesPodNamespaceLabel  = "io.kubernetes.pod.namespace"
 	KubernetesPodUIDLabel        = "io.kubernetes.pod.uid"
 	KubernetesContainerNameLabel = "io.kubernetes.container.name"
+	ContainerdKindLabel          = "io.cri-containerd.kind"
```

Comment:
this circumvents the intent of the CRI because it makes the kubelet aware of the CRI implementation.

the `KubernetesContainerNameLabel` is also intended for deprecation as metrics are sourced from the CRI implementation instead.

see:
https://github.com/kubernetes/kubernetes/blob/20e6883a75db6dbc7908aba2ee69ed9afa8525ed/pkg/kubelet/dockershim/docker_sandbox.go#L625